homeassistant/components/homematicip_cloud/binary_sensor.py

```diff
@@ -104,3 +121,91 @@ def device_class(self):
     def is_on(self):
         """Return true if moisture or waterlevel is detected."""
         return self._device.moistureDetected or self._device.waterlevelDetected
+
+
+class HomematicipSecurityZoneSensorGroup(HomematicipGenericDevice,
+                                         BinarySensorDevice):
+    """Representation of a HomematicIP Cloud water detector."""
+
+    def __init__(self, home, device, post='SecurityZone'):
+        """Initialize heating group."""
+        device.modelType = 'HmIP-{}'.format(post)
+        super().__init__(home, device, post)
+
+    @property
+    def device_class(self):
+        """Return the class of this sensor."""
+        return 'safety'
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the generic device."""
+        attr = super().device_state_attributes
+
+        if self._device.motionDetected:
+            attr.update({ATTR_MOTIONDETECTED: True})
+        if self._device.presenceDetected:
+            attr.update({ATTR_PRESENCEDETECTED: True})
+        from homematicip.base.enums import WindowState
+        if self._device.windowState is not None and \
+                self._device.windowState != WindowState.CLOSED:
+            attr.update({ATTR_WINDOWSTATE: str(self._device.windowState)})
+
+        return attr
+
+    @property
+    def is_on(self):
+        """Return true if security issue detected."""
+        if self._device.motionDetected or \
+                self._device.presenceDetected:
+            return True
+        from homematicip.base.enums import WindowState
+        if self._device.windowState is not None and \
+                self._device.windowState != WindowState.CLOSED:
+            return True
+        return False
+
+
+class HomematicipSecuritySensorGroup(HomematicipSecurityZoneSensorGroup,
+                                     BinarySensorDevice):
+    """Representation of a HomematicIP Cloud water detector."""
+
+    def __init__(self, home, device):
+        """Initialize heating group."""
+        super().__init__(home, device, 'SecuritySensors')
```

Comment:
Please change from "SecuritySensor" just to "Security"

Reply:
Done